src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -53,15 +60,17 @@ class DatabaseTokenRepository implements TokenRepositoryInterface
      * @param  string  $table
      * @param  string  $hashKey
      * @param  int  $expires
+     * @param  int $timeout
      * @return void
      */
     public function __construct(ConnectionInterface $connection, HasherContract $hasher,
-                                $table, $hashKey, $expires = 60)
+                                $table, $hashKey, $expires = 60, $timeout = 60)
```

Comment:
This seems like a breaking change, if anyone extends this class it will throw an error after this is merged. Maybe better to go with this version in the master

Reply:
The child constructor doesn't have to have the same signature as the parent constructor and since the added parameter has a default value it means that there is no breaking change here.

Reply:
Great, I thought they behave like normal method. TIL